napari/_qt/containers/qt_layer_model.py

```diff
@@ -20,7 +20,7 @@ def data(self, index: QModelIndex, role: Qt.ItemDataRole):
         if role == Qt.EditRole:  # used to populate line edit when editing
             return layer.name
         if role == Qt.ToolTipRole:  # for tooltip
-            return layer.name
+            return f'name: {layer.name},  plugin: {layer.source.reader_plugin}'
```

Comment:
hm. The tooltip is a statical part. Why do we use this to serve tooltip instead of call 
https://doc.qt.io/qt-6/qtreewidgetitem.html#setToolTip or https://doc.qt.io/qt-6/qlistwidgetitem.html#setToolTip (i do not remember what is base class)